app/build.gradle

```diff
@@ -4,7 +4,7 @@ android {
     compileSdkVersion 28
     defaultConfig {
         applicationId "it.niedermann.nextcloud.deck"
-        minSdkVersion 14
+        minSdkVersion 15
```

Comment:
Is this mandatory? I'd prefer keeping it at 14, even though it doesn't look _that_ nice on those

Reply:
This is a dependency for the imported library for the color chooser. It wouldn't accept 14 for minSdkVersion. I could see about doing a feature toggle, but that's a foreign concept to me. What would that involve?

Reply:
@stefan-niedermann  so you maybe know, how this exactly works?

Reply:
In theory, one could use `tools:overrideLibrary="com.skydoves.colorpickerpreference"` which will allow to build the app with this library against `minSdk = 14`. This will of course create runtime problems on lower android versions which can have to be caught by using something like

```java
if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.KITKAT){
    // launch the custom color picker
} else{
    // handle api level 14 gracefully (simply do not show the custom color picker here)
}
```

Reply:
I've added the library override, however, Android Studio won't install an SDK with a version as old as 14, so I can't test the version check. Do you know if a way I can manually install an emulator with an SDK of 14?

Reply:
Okay, as far as i know, one can override the min SDK only if the library didn't set a flag to explicitly disallow overrides. :disappointed: 

So, here we go, bump the minSdk to 15. It's fine for me.

Reply:
Yep, i think the impact isn't that big. It's fine.